src/map/pc.c

```diff
@@ -11500,7 +11505,7 @@ bool pc_db_checkid(unsigned int class_)
 		|| (class_ >= JOB_BABY_RUNE      && class_ <= JOB_BABY_MECHANIC2 )
 		|| (class_ >= JOB_SUPER_NOVICE_E && class_ <= JOB_SUPER_BABY_E   )
 		|| (class_ >= JOB_KAGEROU        && class_ <= JOB_OBORO          )
-		|| (class_ >= JOB_REBELLION      && class_ <  JOB_MAX            );
+		|| (class_ >= JOB_REBELLION      || class_ ==  JOB_SUMMONER      );
 }
 
 /**
```

Comment:
Some other notes:

- This commit accidentally re-adds db/const.txt
- There are other places that you might want to look at, to see if they also need to be updated with this, i.e. `pc_check_job_name()`, `itemdb_jobmask2mapid()`, `itemdb_jobid2mapid()`